src/app/services/keybinder/keybinder.service.ts

```diff
@@ -73,6 +74,11 @@ export class KeybinderService {
       () => this.store.dispatch(new queryActions.SendQueryRequestAction(this.activeWindowId)),
       'Send Request'
     );
+    this.bindShortcut(
+      ['Command+,'],
+      () => this.store.dispatch(new windowsMetaActions.ShowSettingsDialogAction({ value: true })),
+      'Show Settings'
+    );
```

Comment:
This wouldn't work since the browsers already defined the shortcut to open *their* settings.